src/main/scala/com/adobe/api/platform/runtime/metrics/EventConsumer.scala

```diff
@@ -54,12 +73,39 @@ case class EventConsumer(settings: ConsumerSettings[String, String], recorders:
   private def processEvent(value: String): Unit = {
     EventMessage
       .parse(value)
+      .map { e =>
+        e.eventType match {
+          case Activation.typeName => activationCounter.increment()
+          case Metric.typeName     => metricCounter.increment()
+        }
+        e
+      }
       .collect { case e if e.eventType == Activation.typeName => e } //Look for only Activations
       .foreach { e =>
         val a = e.body.asInstanceOf[Activation]
         recorders.foreach(_.processEvent(a))
+        updateGlobalMetrics(a)
       }
   }
+
+  private def updateGlobalMetrics(a: Activation): Unit = {
+    a.status match {
+      case Activation.statusSuccess          => statusSuccess.increment()
+      case Activation.statusApplicationError => statusApplicationError.increment()
+      case Activation.statusDeveloperError   => statusDeveloperError.increment()
+      case Activation.statusInternalError    => statusInternalError.increment()
+      case _                                 => //Ignore for now
+    }
+
+    if (a.status != Activation.statusSuccess) statusFailure.increment()
```

Comment:
We could always have the number of failures by adding all failures together `a + b + c` where `a/b/c` are the counts of App, Dev and Internal errors? 
But yeah, this is cleaner.

Reply:
Yes ... consider it like a materialized query where we trade the cost of computing at runtime with space